user-interface/7003-MORD-manage_orders.md

```diff
@@ -1 +1,202 @@
-# Manage orders
\ No newline at end of file
+# Manage orders
+
+Users place orders to describe the trades they would like to make: buy or sell, at what price, how long it is valid for etc. In many cases they can [amend](#amend-order---price) or [cancel](#cancel-orders) these orders while they are still active, for example: changing price.
+
+Once a user has placed an order they may wish to confirm it's [status](https://docs.vega.xyz/docs/mainnet/graphql/enums/order-status) in a [list](#orders-list) of other orders. e.g. whether it has been accepted, filled, how close it is to being filled etc. Users may be interested in the price of their orders relative to the price of the market and how much of the order's size has been filled.
+
+Orders can also be placed on behalf of a user/party via [liquidity](#liquidity-order-shapes) or [pegged](#pegged-order-shapes) order shapes. These order cannot be amended on canceled in the same way as other orders.
+
+Markets also have [statuses](https://docs.vega.xyz/docs/mainnet/graphql/enums/market-state) that may affect how a user perceives the state of an order, e.g if the order was placed while in "normal" continuous trading, but the market is now in auction. 
+
+## Orders list
+
+User will have differing needs/preferences in terms of what they want to see about an order and how orders are grouped and listed. It is common for interfaces to allow users to customize how orders are displayed.
+
+### Field customization
+
+- **should** have the ability to select what [fields/data](#fields) are shown for each order in the list
+- **should** have the ability to change the order of fields (e.g. table columns)
+- **should** have the ability to give each column in the list more or less space
+
+### Fields
+
+When looking at a list of orders, I...
+
+- **must** see [status](https://docs.vega.xyz/docs/mainnet/graphql/enums/order-status) of the order (<a name="7003-MORD-001" href="#7003-MORD-001">7003-MORD-001</a>), and therefore (nested bullets added for context, ACs are on fields that follow)... 
+  - `Active​`
+    - How much of the order is filled / remains unfilled
+    - How close the mark price is to order
+    - If this order was filled at the limit price what would what effect would it have on realized PnL 
+    - I may want to amend or cancel this order
+  - `Expired​`
+    - When did it expire
+    - How much was filled / remaining
+  - `Cancelled​`
+    - What canceled it? When did I cancel it TODO double check that "cancelled" only comes from user action 
```

Comment:
Do you need to do or update this TODO and the one under PARKED?

Reply:
resolved in : https://github.com/vegaprotocol/specs/pull/1334/commits/2ded0c5ae2a5f0dd4975dc46c47202660511d9f7 it doesn't change the fact we should link to the shape when a order is parked